pyproject.toml

```diff
@@ -18,6 +18,7 @@ classifiers = [
 ]
 dependencies = [
     "asdf>=4.0,<5",
+    "asdf-astropy>=0.8.0",
```

Comment:
```suggestion
```

I believe the only indirect usage of asdf-astropy is via gwcs and stdatamodels. Both of those have asdf-astropy pins so may be just remove it here?